portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,27 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
```

Comment:
please order imports alphabetically; see https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing

Reply:
Done